spec.md

```diff
@@ -77,55 +82,59 @@ progressively greater amount of knowledge."
 **Suggestions:**
 - May capture third and fourth depth headings. If it is a long ToC, these are optional.
 
-#### Security
-**Status**: Optional.  
+### Security
+**Status**: Optional.
 **Requirements:**
-- May go here if visibility of security section is important. Otherwise, should be in [Extra Sections](#extra-sections).
+- May go here if it is important to highlight security concerns. Otherwise, it should be in [Extra Sections](#extra-sections).
 
-#### Background
-**Status:** Optional.  
+### Background
+**Status:** Optional.
 **Requirements:**
 - Cover motivation.
 - Cover abstract dependencies.
 - Cover intellecutal provenance: A `See Also` section is also fitting.
 
-#### Install
+### Install
 **Status:** Required by default, optional for doc modules.
 **Requirements:**
 - Code block illustrating how to install.
 
-Subsections:
-- `Dependencies`. Required if necessary.
+**Subsections:**
+- `Dependencies`. Required if there are dependencies.
```

Comment:
How about `if there are unexpected dependencies which must be independently installed`?